### ISSUE TITLE
Bug: Keyboard focus missing from most elements in Firefox

### ISSUE THREAD

**Issue by terichadbourne** [ProtoSchool/protoschool.github.io#351]
In Firefox within an ongoing PR I can see this focus but can't see focus rings when tabbing through lessons, etc. The tabbing jumps over most of the elements it catches in Chrome. 

![image](https://user-images.githubusercontent.com/19171465/73027218-018b0980-3e01-11ea-9663-853a16a53464.png)

In Chrome, tabbing catches all of these: 
![image](https://user-images.githubusercontent.com/19171465/73027270-18316080-3e01-11ea-80b6-2309884787e6.png)
![image](https://user-images.githubusercontent.com/19171465/73027288-20899b80-3e01-11ea-8830-69987956231a.png)
![image](https://user-images.githubusercontent.com/19171465/73027298-27181300-3e01-11ea-952e-2d46d00e5371.png)
![image](https://user-images.githubusercontent.com/19171465/73027325-313a1180-3e01-11ea-87ef-27f6c1c36f96.png)
